lib/v2/chinawriter/index.js

```diff
@@ -0,0 +1,88 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const timezone = require('@/utils/timezone');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const { id } = ctx.params;
+    const limit = ctx.query.limit ? parseInt(ctx.query.limit, 10) : 40;
+
+    const rootUrl = 'http://www.chinawriter.com.cn';
+    const currentUrl = new URL(id ?? '', rootUrl).href;
```

Comment:
```suggestion
    const currentUrl = `${new URL(id ?? '', rootUrl).href}/`;
```
Use the url with trailing hash directly.
https://github.com/DIYgod/RSSHub/actions/runs/6174984526/job/16760843470#step:12:102